src/components/ThreadEnvelope.vue

```diff
@@ -441,6 +441,7 @@ export default {
 <style lang="scss" scoped>
 	.sender {
 		margin-left: 8px;
+		font-weight: bold;
```

Comment:
Sure let's go for it! Just a detail, code indent seems wrong here?

Reply:
Linter doesn't swear

Reply:
We use bold for unread messages. I think this style conflicts.